src/intl/es/page-what-is-ether.json

```diff
@@ -0,0 +1,100 @@
+{
+  "page-what-is-ether-meta-title": "¿Qué es el ether (ETH)? (Una guía completa) | ethereum.org",
```

Comment:
Most of the translations are adding the article "el" ("the") before ether and lowercasing it which is bad IMO.

Source text is: "What is Ether (ETH)?"
Translations should be: "¿Qué es Ether (ETH)?"